src/diffpy/utils/diffraction_objects.py

```diff
@@ -284,6 +278,20 @@ def __rtruediv__(self, other):
 
     @property
     def all_arrays(self):
+        """The 2D array containing `xarray` and `yarray` values.
+
+        Returns
+        -------
+        ndarray
+            The 2D array containing the `xarray` values in q, tth, d, and `yarray`.
+
+        Examples
+        --------
+        >>> my_do.all_arrays[:, 0]  # yarray
```

Comment:
add "To access specific arrays individually, use these slices:"?

Reply:
<img width="753" alt="Screenshot 2024-12-26 at 2 46 56 PM" src="https://github.com/user-attachments/assets/e094a36c-71b9-4b51-ba91-af4d2e0b50e0" />

Yes, great. done

Reply:
There's a stray y-array at the end of the line :(

Reply:
Sorry - fixed now:        

---

            The shape (len(data), 4) 2D array with columns containing the `yarray` (intensity)
            and the `xarray` values in q, tth, and d.